notes.md

```diff
@@ -0,0 +1,63 @@
+MAKING A MOVIE APP
```

Comment:
Creating blueprint of how your app will look like is definitely an important task. I love it!